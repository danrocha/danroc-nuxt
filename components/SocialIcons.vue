<template>
  <ul class="flex space-x-2 text-lg text-gray-800">
    <li>
      <a href="//twitter.com/danrocc" target="_blank" rel="nofollow"
        ><svg-icon icon="twitter"
      /></a>
    </li>
    <li>
      <a href="//github.com/danrocha" target="_blank" rel="nofollow"
        ><svg-icon icon="github"
      /></a>
    </li>
    <li>
      <a href="//linkedin.com/in/danrocha" target="_blank" rel="nofollow"
        ><svg-icon icon="linkedin"
      /></a>
    </li>
  </ul>
</template>
