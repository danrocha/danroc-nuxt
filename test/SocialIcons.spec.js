import { mount, RouterLinkStub } from '@vue/test-utils'
import SocialIcons from '../components/SocialIcons'

describe('SocialIcons', () => {
  it('Render correctly', () => {
    const wrapper = mount(SocialIcons, {
      stubs: {
        NuxtLink: RouterLinkStub,
      },
    })
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
