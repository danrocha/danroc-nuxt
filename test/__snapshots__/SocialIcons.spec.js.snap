// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialIcons Render correctly 1`] = `
<ul
  class="flex space-x-2 text-lg text-gray-800"
>
  <li>
    <a
      href="//twitter.com/danrocc"
      rel="nofollow"
      target="_blank"
    >
      <svg
        class="icon"
      >
        <use
          xlink:href="#twitter"
        />
      </svg>
    </a>
  </li>
   
  <li>
    <a
      href="//github.com/danrocha"
      rel="nofollow"
      target="_blank"
    >
      <svg
        class="icon"
      >
        <use
          xlink:href="#github"
        />
      </svg>
    </a>
  </li>
   
  <li>
    <a
      href="//linkedin.com/in/danrocha"
      rel="nofollow"
      target="_blank"
    >
      <svg
        class="icon"
      >
        <use
          xlink:href="#linkedin"
        />
      </svg>
    </a>
  </li>
</ul>
`;
